SeleniumWebDriverAndLocators/SeleniumWebDriverAndLocators/YandexMailTests.cs

```diff
@@ -0,0 +1,283 @@
+            //this.driver.FindElement(By.Name("lastName")).SendKeys("WebDriver");
+            //this.driver.FindElement(By.Name("Username")).Clear();
+            //this.driver.FindElement(By.Name("Username")).SendKeys("Selenium2Web2Driver");
+            //this.driver.FindElement(By.Name("Passwd")).SendKeys("$elenium789");
+            //this.driver.FindElement(confirmationField).SendKeys("$elenium789");
+            //this.driver.FindElement(By.XPath("//span[text()='Далее']")).Click();
+
+            //IsElementVisible(furtherButton);
+            //this.driver.FindElement(furtherButton).Click();
+
+            //IsElementVisible(phoneNumberField);
+            //this.driver.FindElement(phoneNumberField).SendKeys("+*** *********");
+            //this.driver.FindElement(By.XPath("//button[@class='VfPpkd-LgbsSe VfPpkd-LgbsSe-OWXEXe-k8QpJ VfPpkd-LgbsSe-OWXEXe-dgl2Hf nCP5yc AjY5Oe DuMIQc qIypjc TrZEUc lw1w4b']")).Click();
+
+            //IsElementVisible(codeField);
+            //this.driver.FindElement(codeField).SendKeys("******");
+            //this.driver.FindElement(By.XPath("//button[@class='VfPpkd-LgbsSe VfPpkd-LgbsSe-OWXEXe-k8QpJ VfPpkd-LgbsSe-OWXEXe-dgl2Hf nCP5yc AjY5Oe DuMIQc qIypjc TrZEUc lw1w4b']")).Click();
+
+            //IsElementVisible(furtherButton);
+            //this.driver.FindElement(By.Id("day")).SendKeys("1");
+            //this.driver.FindElement(By.Id("month")).Click();
+            //this.driver.FindElement(By.XPath("//select/option[text()='Январь']")).Click();
+            //this.driver.FindElement(By.Id("year")).SendKeys("2000");
+            //this.driver.FindElement(By.XPath("//select/option[text()='Женский']")).Click();
+            //this.driver.FindElement(furtherButton).Click();
+
+            //IsElementVisible(skipButton);
+            //this.driver.FindElement(skipButton).Click();
+
+            //IsElementVisible(acceptButton);
+            //this.driver.FindElement(acceptButton).Click();
+
+            //IsElementVisible(furtherButton);
+            //this.driver.FindElement(acceptButton).Click();
+
+            //Thread.Sleep(15000);
+
+            //IsElementVisible(furtherButton);
+            //this.driver.FindElement(furtherButton).Click();
+
+            //IsElementVisible(furtherButton);
+            //this.driver.FindElement(furtherButton).Click();
+
+            //IsElementVisible(checkTextField);
+            //Thread.Sleep(10000);
+            //this.driver.FindElement(furtherButton).Click();
+        }
+
+        [Test]
+        public void GetLoggedInUser()
+        {
+            //Arrange
+            LoginToMailBox();
+
+            string expectedUserName = "Selenium1Web1Driver";
+
+            //Act
+            var actualUserName = this.driver.
+                FindElement(By.XPath("//a[@class='user-account user-account_left-name user-account_has-ticker_yes user-account_has-accent-letter_yes count-me legouser__current-account legouser__current-account i-bem']/span[1]")).
```

Comment:
It is unclear which button is pressed by xPath. Please define the locator as a separate parameter. 
`//Arrange`
`var accountButton = By.XPath("//a[contains(@class,'user-account')]");`

`//Act`
`var actualUserName = this.driver.FindElement(accountButton).Text;`